Build/Build.csproj

```diff
@@ -1,5 +1,6 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
 		<TargetFramework>net472</TargetFramework>
+		<LangVersion>8.0</LangVersion>
```

Comment:
We should define 9.0 since only 9.0 allows you writing generic code with NRT.

https://github.com/dotnet/csharplang/blob/master/proposals/csharp-9.0/unconstrained-type-parameter-annotations.md

Reply:
Is this project used for something other than aggregation of build related files inside IDE? @ig-sinicyn ?